sysconfigs/daint/upstreams.yaml

```diff
@@ -1,7 +1,7 @@
 upstreams:
   base:
-    install_tree: /project/g110/spack/upstream/daint_v0.18.1.12/base
+      install_tree: /project/g110/spack/upstream/tmp_dev0.20.1/base
```

Comment:
Does the "tmp_" stay?

Reply:
No I change it at the very end.
Otherwise testing will rebuild all deps